engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -330,6 +330,11 @@ class K8sDeploymentManager(
     }
   }
 
+  override def getDeploymentStatusesToUpdate: Future[Map[newdeployment.DeploymentId, DeploymentStatus]] = {
+    // We don't handle this flow for k8s dm, so we return empty list map of updates
+    Future.successful(Map.empty)
```

Comment:
Why on the k8d there are no statuses to update? How do we get information about state in this case?

Reply:
I made it more explicit - I extracted a `DeploymentSynchronisationSupport` interface with `NoDeploymentSynchronisationSupport` implementation. I also added comments why it is not supported by this DM